osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/Components/PathControlPointConnectionPiece.cs

```diff
@@ -20,7 +20,17 @@ public class PathControlPointConnectionPiece : CompositeDrawable
 
         private readonly Path path;
         private readonly Slider slider;
-        private readonly int controlPointIndex;
+        private int controlPointIndex;
+
+        public int ControlPointIndex
+        {
+            get => controlPointIndex;
+            set
+            {
+                controlPointIndex = value;
+                updateConnectingPath();
+            }
+        }
```

Comment:
maybe make this a bindable to match `sliderPosition`/`pathVersion` at this point?

Reply:
Turns out the `updateConnectingPath` call wasn't necessary there since the pathVersion value changes right after so I made it an auto-implemented property instead.